actions/test/run_local.ts

```diff
@@ -25,9 +25,24 @@ const main = async () => {
   // Get provider
   const provider = await getProvider(testRuntime.context, chainId);
 
-  // Run one of the 2 Execution modes (single block, or watch mode)
+  // Run one of the 3 Execution modes:
+  //  - Single tx: Process just one tx
+  //  - Single block: Process just one block
+  //  - Watch mode: Watch for new blocks, and process them
+  const txEnv = process.env.TX;
   const blockNumberEnv = process.env.BLOCK_NUMBER;
-  if (blockNumberEnv) {
+  if (txEnv) {
+    // Execute once, for a specific tx
+    console.log(
+      `[run_local] Processing ONCE for a specific transaction: ${txEnv}...`
+    );
+
+    await processTx(provider, txEnv, chainId, testRuntime).catch(() => {
+      exit(101);
```

Comment:
Why exit code `101`? Is this a standard or is this defined anywhere? (Sorry, not aware)

Reply:
haha, no. Just to make it different than 100 I used for the other. I will just define consts for them